tests/shared/nlu/training_data/test_training_data.py

```diff
@@ -20,7 +28,7 @@
 
 
 def test_luis_data():
-    td = load_data("data/examples/luis/demo-restaurants_v5.json")
+    td = load_data("data/examples/luis/demo-restaurants_v7.json")
```

Comment:
can we please add tests for the roles? And for regex features?

Reply:
Done!